docs/release-notes/8.6.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[release-notes-header-8.6.0]]
+== 8.6
+
+[discrete]
+[[release-notes-8.6.0]]
+=== 8.6.0
+
+[discrete]
+[[known-issue-8.6.0]]
+==== Known issues
+There are no known issues in 8.6.0.
+
+[discrete]
+[[breaking-changes-8.6.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
```

Comment:
This attribute definition needs to use the full URL; an attribute definition _inside_ a snippet can't reference another attribute _outside_ the snippet, due to the vagaries of how snippets and scoping work in Asciidoc.

Even if the snippet doesn't currently contain any RNs or content, it'd still be good to make sure that the attribute is correctly defined in case we add content later on.

```suggestion
:pull: https://github.com/elastic/kibana/pull/
```